src/main/java/org/takes/facets/auth/codecs/CcAES.java

```diff
@@ -145,15 +118,40 @@ public Identity decode(final byte[] bytes) throws IOException {
     }
 
     /**
+     * This method will create AlgorithmParameterSpec with the block size.
+     * @return The AlgorithmParameterSpec
+     */
+    private static AlgorithmParameterSpec initAlgorithmParameterSpec() {
+        final SecureRandom random = new SecureRandom();
+        final byte[] bytes = new byte[CcAES.BLOCK];
+        random.nextBytes(bytes);
+        return new IvParameterSpec(bytes);
+    }
+
+    /**
+     * This method to check the block size of the key.
```

Comment:
@sebing can you write `Checks the block...` instead of verbose `This method to check the block...`?